CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Nothing yet
 * [View Full Diff](https://github.com/westonganger/protected_attributes_continued/compare/v1.7.0...master)
 
+## 1.8.0 - 2021-01-27
+
+* [#21](https://github.com/westonganger/protected_attributes_continued/pull/21) - Fix bug when calling `first_or_create` on an association relation.
```

Comment:
Could you just add this under the Unreleased section?

Reply:
Moved this under **Unreleased**. Let me know if I did it right.